src/Lecture2.hs

```diff
@@ -178,7 +213,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing (x:xs:xss)
+    | xs < x    = False
+    | otherwise = isIncreasing (xs:xss)
```

Comment:
Absolutely correct solution 👍🏻 

You can make it a bit shorter by relying on Haskell laziness 🦥 

```suggestion
isIncreasing (x:xs:xss) = x < xs && isIncreasing (xs : xss)
```

Reply:
Wow, looks good